daemons/ipa-sam/ipa_sam.c

```diff
@@ -283,6 +287,62 @@ static bool is_null_sid(const struct dom_sid *sid)
 	return true;
 }
 
+static int dom_sid_string_buf(const struct dom_sid *sid, char *buf, int buflen)
```

Comment:
Please rename to `_domain_sid_string_buf` to avoid possible conflicts when loading in the same process.